cluster/src/test/scala/io/snappydata/cluster/JDBCConnectionPoolTestSuite.scala

```diff
@@ -60,6 +59,33 @@ class JDBCConnectionPoolTestSuite extends SnappyFunSuite with BeforeAndAfterAll
     conn1.close()
   }
 
+  test("GetTypeInfo-Boolean-Test") {
```

Comment:
Better to move the test to BugTest.scala

Reply:
Moved